File: Numbers/Digit_At/digit_at.go
package main

import (
	"fmt"
	"math"
)

func main() {

	fmt.Println(digitAt(1234567, 1) == 7)
	fmt.Println(digitAt(1234567, 2) == 6)
	fmt.Println(digitAt(1234567, 3) == 5)
	fmt.Println(digitAt(1234567, 4) == 4)

}

func digitAt(num, index int) int {
	return (num / int(math.Pow(10, float64(index-1)))) % 10
}

File: Numbers/Digit_At/digit_at.py

def digit_at(number, index):
    return number // 10 ** (index - 1) % 10


print(digit_at(1234567, 1) == 7)
print(digit_at(1234567, 2) == 6)
print(digit_at(1234567, 3) == 5)
print(digit_at(1234567, 4) == 4)
